app/model/orm/orm_base.py

```diff
@@ -20,7 +20,17 @@ def _asdict(self):
         }
 
     def __str__(self):
-        return f"<{type(self).__name__} {self._asdict()}>"
+        parts = []
+
+        if hasattr(self, 'id'):
+            parts.append(f"id={self.id}")
+        if hasattr(self, 'publicId'):
+            parts.append(f"publicId={self.publicId}")
+
+        return f"<{type(self).__name__} {', '.join(parts)}>"
```

Comment:
Flask-admin uses `str(` to format objects, so let's simplify the output and print the full object contents in `__repr__`